<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Zoom Out</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="minimum">
       <number>-180</number>
      </property>
      <property name="maximum">
       <number>184</number>
      </property>
      <property name="singleStep">
       <number>0</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Zoom In</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignTop">
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="maximumSize">
       <size>
        <width>99</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_BMP"/>
   </widget>
   <widget class="QMenu" name="menuBVP">
    <property name="title">
     <string>BVP</string>
    </property>
    <addaction name="actionBVP"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBVP"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_BMP">
   <property name="text">
    <string>Save BMP</string>
   </property>
  </action>
  <action name="actionBVP">
   <property name="text">
    <string>BVP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
